### ISSUE TITLE
Is it possible to add Google Play Games Services to this template?

### ISSUE THREAD

**Issue by JustYourAverageJosh** [Shin-NiL/godot-custom-mobile-template#4]
Hi, I recently setup Admob using this premade android template, works great! I was hoping to also use google play game services as well, but I am new to coding and am not sure how to compile Godot for Android.

This module claims to work with your admob module in Godot 3.1: 
https://github.com/mani144/gpgs 

Would you be able/willing to make a custom android template that also has Google Play Games Services as well?

**Comment by Shin-NiL** [comment]
Hello there!

Unfortunately it's not possible as to use the GPGS you need to put some values on the android manifest file, wich can only be done before the compilation.

With Godot 3.2 and the [new Android plugin system](https://godotengine.org/article/godot-3-2-will-get-new-android-plugin-system) things like this will be easier.

**Comment by FlavioFS** [comment]
@JustYourAverageJosh 

Exactly! I struggled with this for 2 weeks, since there are many (sometimes undocumented) caveats in the process like:
 - Link your app to Firebase, download *google-services.json* from there and put it into *GODOT_ROOT/platforms/android/java*
 - Modify config.py by adding your **Play Games Services ID** before compiling
 - Sometimes (if API is outdated), update *compileSdkVersion* and *buildToolsVersion*, and replace "compile" with "implementation" in *GODOT_ROOT/platforms/android/java/builde.gradlew* (this is usually not needed).
 - And the list goes on...

It was so specific I've documented the process in [**this sample repo**](https://github.com/FlavioFS/godot-clef-clicker).

There were also other minor issues like:  
 - FrogSquare's API (the one GPGS I've used) had Python indentation mistakes at config.py

Sadly, as Shin said, it is not possible to pre-compile it yet.
